src/lib.rs

```diff
@@ -196,6 +196,11 @@ feature! {
 #[allow(missing_docs)]
 pub mod unistd;
 
+#[cfg(any(feature = "poll", feature = "event"))]
+mod timeout;
+#[cfg(any(feature = "poll", feature = "event"))]
+pub use timeout::*;
```

Comment:
`nix::Timeout` is too generic.  There are plenty of other functions with timeouts that don't use the same data type.  For example, `aio_suspend` and `recvmmsg`.  Instead, can you nest it under the `poll` and `epoll` namespaces?  It's ok if it's the same type, as long as it's not exported from the crate's root.

Reply:
Keeping the `mod timeout` private, re-exporting them in `src/poll.rs` and `src/sys/epoll.rs` would do the trick.

BTW, it might be better to rename `timeout` to `poll_timeout` given it is not that generic

Reply:
I've implemented this.